include/onnxruntime/core/common/common.h

```diff
@@ -95,6 +99,57 @@ void LogRuntimeError(uint32_t session_id, const common::Status& status, const ch
 #define ORT_WHERE_WITH_STACK \
   ::onnxruntime::CodeLocation(__FILE__, __LINE__, __PRETTY_FUNCTION__, ::onnxruntime::GetStackTrace())
 
+#ifdef ORT_NO_EXCEPTIONS
+
```

Comment:
Assuming you're adding a CMake option to set this (even if the change to completely disable exceptions in the build is added later). Should that be in this PR?

Reply:
I will keep this for now, may remove it in before merge



---
In reply to: [475423597](https://github.com/microsoft/onnxruntime/pull/4894#discussion_r475423597) [](ancestors = 475423597)